subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junit5/JUnitPlatformIntegrationTest.groovy

```diff
@@ -0,0 +1,29 @@
+package org.gradle.testing.junit5
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.integtests.fixtures.DefaultTestExecutionResult
+import org.gradle.integtests.fixtures.TestResources
+import org.junit.Rule
+import spock.lang.Timeout
+
+class JUnitPlatformIntegrationTest extends AbstractIntegrationSpec {
+    @Rule
+    final TestResources resources = new TestResources(testDirectoryProvider)
+
+    def setup() {
+        executer.noExtraLogging()
+    }
+
+    @Timeout(10)
```

Comment:
Right, it should be fast. Since the test hangs indefinitely I just wanted it to fail if it wasn't going to complete normally. This wouldn't stick around in a working impl.

Reply:
Thanks for clarifying.